_blueprints/model_overview.adoc

```diff
@@ -3,25 +3,30 @@
 [graphviz, model_overview, svg]
 ----
 digraph G {
-	IDENTITY -> USER
+	USER -> LOGIN
 	
-	IDENTITY -> TEAM_ROLE
-	USER -> WORKITEM[label="CRUD|Assigned"]
+	USER -> TEAM_ROLE
+	LOGIN -> WORKITEM[label="CUD|Assigned"]
 	
 	TEAM_ROLE -> TEAM
 	TEAM_ROLE -> PERMISSION
 
 	TEAM -> PROJECT
-	
-	
+	ORGANIZATION -> TEAM
```

Comment:
Are there not teams associated with projects? Contributors to "almighty-core", for example?